src/components/Icon.tsx

```diff
@@ -359,6 +359,9 @@ export const Icons = {
     <path d="M21 3H14C13.229 3 12.532 3.301 12 3.78C11.468 3.301 10.771 3 10 3H3C2.447 3 2 3.448 2 4V19C2 19.553 2.447 20 3 20H8.758C9.284 20 9.8 20.214 10.172 20.586L11.293 21.707C11.302 21.716 11.314 21.719 11.323 21.728C11.409 21.807 11.505 21.877 11.617 21.924C11.618 21.924 11.618 21.924 11.619 21.924C11.74 21.974 11.869 22 12 22C12.131 22 12.26 21.974 12.381 21.924C12.382 21.924 12.382 21.924 12.383 21.924C12.495 21.877 12.591 21.807 12.677 21.728C12.686 21.719 12.698 21.716 12.707 21.707L13.828 20.586C14.2 20.214 14.716 20 15.242 20H21C21.553 20 22 19.553 22 19V4C22 3.448 21.553 3 21 3ZM8.758 18H4V5H10C10.552 5 11 5.449 11 6V18.689C10.342 18.246 9.557 18 8.758 18ZM20 18H15.242C14.443 18 13.658 18.246 13 18.689V6C13 5.449 13.448 5 14 5H20V18Z" />
   ),
   // Misc
+  criticalPath: (
+    <path d="M5.41663 16.666H12.37C12.6666 17.2935 13.03 17.851 13.4108 18.3327H5.41663C3.34913 18.3327 1.66663 16.6502 1.66663 14.5827C1.66663 12.5152 3.34913 10.8327 5.41663 10.8327H11.25C11.9391 10.8327 12.5 10.2718 12.5 9.58268C12.5 8.89352 11.9391 8.33268 11.25 8.33268H6.07079C6.47579 7.86852 6.87329 7.30852 7.19913 6.66602H11.25C12.8583 6.66602 14.1666 7.97435 14.1666 9.58268C14.1666 11.191 12.8583 12.4993 11.25 12.4993H5.41663C4.26746 12.4993 3.33329 13.4335 3.33329 14.5827C3.33329 15.7318 4.26746 16.666 5.41663 16.666ZM1.66663 4.16602C1.66663 2.78768 2.78829 1.66602 4.16663 1.66602C5.54496 1.66602 6.66663 2.78768 6.66663 4.16602C6.66663 6.82185 4.16663 8.33268 4.16663 8.33268C4.16663 8.33268 1.66663 6.82268 1.66663 4.16602ZM2.91663 4.16602C2.91663 4.85602 3.47663 5.41602 4.16663 5.41602C4.85663 5.41602 5.41663 4.85602 5.41663 4.16602C5.41663 3.47602 4.85663 2.91602 4.16663 2.91602C3.47663 2.91602 2.91663 3.47602 2.91663 4.16602ZM13.3333 14.166C13.3333 12.7877 14.455 11.666 15.8333 11.666C17.2116 11.666 18.3333 12.7877 18.3333 14.166C18.3333 16.8218 15.8333 18.3327 15.8333 18.3327C15.8333 18.3327 13.3333 16.8227 13.3333 14.166ZM14.5833 14.166C14.5833 14.856 15.1433 15.416 15.8333 15.416C16.5233 15.416 17.0833 14.856 17.0833 14.166C17.0833 13.476 16.5233 12.916 15.8333 12.916C15.1433 12.916 14.5833 13.476 14.5833 14.166Z" />
```

Comment:
@JonnCharpentier - it doesn't look like this icon is centered within the 24x24 viewbox. Can we get that re-exported  to properly to be centered within?
<img width="135" alt="Screenshot 2023-05-17 at 10 52 44 AM" src="https://github.com/homebound-team/beam/assets/1143861/1e5ae22e-b772-48a1-a7d3-48871f5f3aa3">

Reply:
@bdow the icon is exported from here, and is centered 
<img width="530" alt="image" src="https://github.com/homebound-team/beam/assets/92122357/0a8eab53-a43b-4aea-a772-c03f55917282">

Reply:
I believe  I seen this scenario for many icons, being 20x20